lib/6to5/generation/whitespace.js

```diff
@@ -11,15 +11,17 @@ Whitespace.prototype.getNewlinesBefore = function (node) {
   var startToken;
   var endToken;
   var tokens = this.tokens;
+  var token;
+
+  for (var i = 0; i < tokens.length; i++) {
```

Comment:
My use case is running 6to5 in Node via Webpack so that's what I'm optimizing for :-)